types/tools/block-tool.d.ts

```diff
@@ -117,11 +117,16 @@ export interface BlockToolConstructable extends BaseToolConstructable {
    */
   conversionConfig?: ConversionConfig;
 
-  /**
+  /** 
    * Is Tool supports read-only mode, this property should return true
    */
   isReadOnlySupported?: boolean;
 
+  /**
+  * Is Tool handles the default event explicitly,this property should return true 
```

Comment:
Please provide a more easy-understandable description of the use case of this option.